### DISCUSSION TITLE
Does Clink support Powershell?

### DISCUSSION THREAD

**Discussion by haminnhtri5555** [chrisant996/clink#277] [Ideas]
So yeh, does Clink support Powershell?

**Comment by chrisant996** [comment]
No, and Powershell already uses Readline (PSReadline, a spiritual variant of Readline if not an actual variant) and has integrated argument completion and input line coloring and customizable prompt and key bindings, etc. All of that would stop working if Clink were injected into Powershell. There are many drawbacks and so little benefit in exchange, so it's not something I would even attempt to enable (if it's even technically possible, which I'm not sure of).
